src/isomorphic/React.js

```diff
@@ -126,10 +126,12 @@ if (__DEV__) {
       get: function() {
         lowPriorityWarning(
           warnedForCreateClass,
-          'React.createClass is no longer supported. Use a plain JavaScript ' +
-            "class instead. If you're not yet ready to migrate, " +
-            'create-react-class is available on npm as a temporary, ' +
-            'drop-in replacement.',
+          'Accessing createClass via the main React package is deprecated,' +
+            ' and will be removed in React v16.0.' +
+            " Use a plain JavaScript class instead. If you're not yet " +
+            'ready to migrade, create-react-class v15.5.3 is available ' +
```

Comment:
Yes, probably. I just mean that we shouldn't give the impression that 15.5.3 is specific version to try because we know it's already broken (for AMD). We should recommend that they use the latest available 15.x version of create-react-class.